drivers/hmis_csv_twenty_twenty_six/app/models/hmis_csv_twenty_twenty_six.rb

```
@@ -40,28 +47,119 @@ def self.importable_files_map
     }.freeze
   end
 
+  def self.custom_files_config
+    @custom_files_config ||= begin
+      custom_dir = Rails.root.join('drivers', 'hmis_csv_twenty_twenty_six', 'config', 'custom')
+
+      if Dir.exist?(custom_dir)
+        all_custom_files = []
+
+        # Load all YAML files from the custom directory
+        Dir.glob(File.join(custom_dir, '*.yaml')).sort.each do |config_file|
+          config = YAML.load_file(config_file, permitted_classes: [Date, Time])
+          if config && config['custom_files'].is_a?(Array)
+            all_custom_files.concat(config['custom_files'])
+          else
+            Rails.logger.warn "Custom file config #{config_file} has invalid structure - expecting 'custom_files' array"
+          end
+        rescue StandardError => e
+          Rails.logger.error "Failed to load custom file config #{config_file}: #{e.message}"
+          raise e if Rails.env.development? || Rails.env.test?
```

Just to confirm, are we sure it's okay to swallow missing-file errors in production? (same concern on 108, 132, and 149)

---

I believe that's expected.  The import should not fail if the custom config files aren't there or are malformed.  It should be clear in the importer what was imported.